.style.yapf

```diff
@@ -0,0 +1,2 @@
+[style]
+based_on_style = google
```

Comment:
Seeing the nature of this file, I assume it does not need a copyright banner?

Reply:
I think the IP is firmly at 0%. heh